services/command-generate-config-getter.js

```diff
@@ -9,7 +9,6 @@ const OPTIONS_DATABASE_MANDATORY = [
   'dbPassword',
 ];
 const OPTIONS_DATABASE_OPTIONAL = [
-  'dbSchema',
```

Comment:
This should not be removed now (see comment below). Prompt are preserved for _now_ (no breaking change in first iteration).